packages/cashier/src/Components/Error/__tests__/withdrawal-locked.spec.js

```diff
@@ -0,0 +1,140 @@
+        const need_poi_account_status = setAccountStatus('none', '', '');
+
+        const wrapper = render(
+            <Router history={history}>
+                <WithdrawalLocked account_status={need_poi_account_status} is_10K_limit />
+            </Router>
+        );
+
+        const btn = wrapper.container.querySelector('.dc-checklist__item-status--action');
+        fireEvent.click(btn);
+
+        expect(history.location.pathname).toBe(routes.proof_of_identity);
+        expect(screen.getByText('Upload a proof of identity to verify your identity')).toBeInTheDocument();
+    });
+
+    it('Should show "Check proof of address document verification status" message and redirect to account/proof_of_address when "-->" button clicked', () => {
+        const history = createBrowserHistory();
+        const need_poa_account_status = setAccountStatus('', 'pending', 'document');
+
+        const { container } = render(
+            <Router history={history}>
+                <WithdrawalLocked account_status={need_poa_account_status} is_10K_limit />
+            </Router>
+        );
+
+        const node_list = container.querySelectorAll('.dc-checklist__item-status--action');
+        Array.from(node_list).map(node => fireEvent.click(node));
+
+        expect(history.location.pathname).toBe(routes.proof_of_address);
+        expect(screen.getByText('Check proof of address document verification status')).toBeInTheDocument();
+    });
+
+    it('Should show "Upload a proof of address to verify your address" message and redirect to account/proof_of_address when "-->" button clicked', () => {
+        const history = createBrowserHistory();
+        const need_poa_account_status = setAccountStatus('', 'none', 'document');
+
+        const { container } = render(
+            <Router history={history}>
+                <WithdrawalLocked account_status={need_poa_account_status} is_10K_limit />
+            </Router>
+        );
+
+        const node_list = container.querySelectorAll('.dc-checklist__item-status--action');
+        Array.from(node_list).map(node => fireEvent.click(node));
+
+        expect(history.location.pathname).toBe(routes.proof_of_address);
+        expect(screen.getByText('Upload a proof of address to verify your address')).toBeInTheDocument();
+    });
+
+    it('Should show "Complete the financial assessment form" message and redirect to account/financial_assessment when "-->" button clicked', () => {
+        const history = createBrowserHistory();
+        const account_status = setAccountStatus('', '', '');
+
+        const { container } = render(
+            <Router history={history}>
+                <WithdrawalLocked account_status={account_status} is_10K_limit is_ask_financial_risk_approval />
+            </Router>
+        );
+
+        const node_list = container.querySelectorAll('.dc-checklist__item-status--action');
+        Array.from(node_list).map(node => fireEvent.click(node));
```

Comment:
As you will have only one checklist item you can use your previous approach to fire the button.
```
const btn = wrapper.container.querySelector('.dc-checklist__item-status--action');
fireEvent.click(btn);
```